config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -42,6 +42,8 @@ presubmits:
         - --timeout=105
         - --scenario=kubernetes_e2e
         - --
+        # cap bazel's JVM memory usage to resources.requests.memory - 2Gi
+        - --inject-bazelrc=startup --host_jvm_args=-Xmx4g
```

Comment:
has this been tested?

Reply:
I tested this by modifying my .bazelrc locally.  Capping to 2g caused excessive garbage collection